.travis.yml

```diff
@@ -38,10 +51,16 @@ script:
   # kube-addon-manager is responsible for managing other kubernetes components, such as kube-dns, dashboard, storage-provisioner..
   #- JSONPATH='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status};{end}{end}'; until kubectl -n kube-system get pods -lcomponent=kube-addon-manager -o jsonpath="$JSONPATH" 2>&1 | grep -q "Ready=True"; do sleep 1;echo "waiting for kube-addon-manager to be available"; kubectl get pods --all-namespaces; done
   - make image
-  #- ./ci/travis-ci.sh
-after_success:
   - make push
+  #- ./ci/travis-ci.sh
+  # If this build is running due to travis release tag, and
+  # this job indicates to push the release downstream, then
+  # go ahead and tag the dependent repo.
+  - if [ ! -z $TRAVIS_TAG ] && [ $RELEASE_TAG_DOWNSTREAM = 1 ] && [ "$TRAVIS_REPO_SLUG" == "openebs/jiva-operator" ]; then
```

Comment:
isn't TRAVIS_REPO_SLUG always eq to openebs/jiva-operator?

Reply:
Should env RELEASE_TAG_DOWNSTREAM=0 by default

Reply:
For forked repo - the TRAVIS_SLUG will be different and we don't want to do downstream based on some user / forked repo getting tagged. 

By default we need to tag the downstream if this repo is tagged. Otherwise we can use them together.